.github/workflows/assign_reviewer.yml

```
@@ -0,0 +1,30 @@
+name: Assign Reviewer
+
+on:
+  pull_request:
+    types: ["labeled"]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  specific_review_requested:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.label.name == 'dependencies'}}
+    steps:
+    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+    - name: Pick assignee
+      id: pick-assignee
+      uses: ./.github/actions/pick_assignee
+    - name: Assign reviewer
+      env:
+        GH_TOKEN: ${{ github.token }}
+        PR: ${{ github.event.pull_request.number }}
+        ASSIGNEE: ${{ steps.pick-assignee.outputs.assignee }}
+      run: |
+        gh api \
+          --method POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          "/repos/edgelesssys/constellation/pulls/$PR/requested_reviewers" \
+          -f "reviewers[]=$ASSIGNEE"
```

```suggestion
          "/repos/edgelesssys/constellation/pulls/${PR}/requested_reviewers" \
          -f "reviewers[]=${ASSIGNEE}"
```
Pretty sure this is what the CI wants

---

It actually wanted quotes, but this is also a good suggestion :)